Only available on aarch64 and x86_64, build fails on other architectures.

--- a/main/CMakeLists.txt
+++ b/main/CMakeLists.txt
@@ -9,6 +9,3 @@ if (WIN32)
     add_subdirectory(forwarder)
 endif ()
 
-if (NOT EMSCRIPTEN)
-    add_subdirectory(updater)
-endif ()
\ No newline at end of file
